app/views/devise/registrations/dashboard.html.erb

```diff
@@ -2,3 +2,14 @@
 <div class="page-header">
   <h1><%= t('dashboard.title', name: @user.first_name) %></h1>
 </div>
+
+<% if user_signed_in? and current_user.non_single_teams.present? %>
```

Comment:
Why do you check for `user_signed_in` although the user is redirected to a 40x page anyways if not signed in and trying to access the dashboard?